### PR TITLE
Extend data request model

### PR DESCRIPTION
**Pull request by dvenckusuchgo**
New fields added for allowing multiple explorers.

- filter_source
- filter_source_internal_id

### CODE REVIEW

**Comment by dvenckusuchgo** [comment]
amanuensis:/pyproject.toml references userPortalDataModel v1.1.0 (in anticipation of a new release integrating this branch)

File: userportaldatamodel/schemas/search_schema.py
Comment (bobaekang) [inline userportaldatamodel/schemas/search_schema.py:8]: Just curious, but why is this commented out bit is left around?

Comment (dvenckusuchgo) [reply userportaldatamodel/schemas/search_schema.py:8]: I was attempting to include serialization and ran into obstacles.  I left it in place to show what remains to be done so someone else can pickup where I left off, but I can just as easily remove it.

Comment (bobaekang) [reply userportaldatamodel/schemas/search_schema.py:8]: If that's the case, can you please describe exactly what is to be done in comment? I'm not as familiar with SQLAlchemy works as you do, and your description will be a great help to whoever will come to pick it up. I'm afraid that commented out lines of code just looks like something to be removed without such description.

Comment (dvenckusuchgo) [reply userportaldatamodel/schemas/search_schema.py]: Additional details added to the comment and I removed necessary lines.  

Please note, this PR needs to be merged & a new release generated before amanuensis PR can be merged in (I need to the update the amanuensis poetry files with the new version of userPortalDataModel)

Comment (bobaekang) [reply userportaldatamodel/schemas/search_schema.py]: Thank you for adding the comment, and my apologies for dragging this review too long. Please feel free to move forward.

**Review by dvenckusuchgo — COMMENTED** [review]
All changes made. Awaiting approval

**Comment by dvenckusuchgo** [comment]
Revise setup.py and change version to 1.1.1

**Review by tianj7 — APPROVED** [review]

**Pull request merged**
